crates/edr_op/src/eip1559.rs

```diff
@@ -1,13 +1,19 @@
 use edr_eip1559::ConstantBaseFeeParams;
 use edr_primitives::Bytes;
 
+/// Holocene version of the dynamic base fee parameter.
+pub const HOLOCENE_BASE_FEE_PARAM_VERSION: u8 = 0x0;
 /// Currently used version of the dynamic base fee parameter.
```

Comment:
Suggestion(non-blocking)
```suggestion
/// Post-Jovian version of the dynamic base fee parameter.
```